src/components/editMeasure/measureDetails/measureMetadata/MeasureMetadata.tsx

```diff
@@ -36,6 +37,9 @@ export default function MeasureMetadata(props: MeasureMetadataProps) {
   const measureServiceApi = useMeasureServiceApi();
   const [success, setSuccess] = useState<string>("");
   const [error, setError] = useState<string>("");
+  const { getUserName } = useOktaTokens();
+  const userName = getUserName();
+  const canEdit = userName === measure.createdBy ? true : false;
```

Comment:
I'm liking this pattern with `canEdit`. Do you think it would work in `MeasureInformation` too?

Reply:
@jkotanchik-SB It would. I'll change it.

Reply:
@jkotanchik-SB I made the change.